e2e/cypress/integration/ckan_extensions.spec.js

```diff
@@ -23,14 +25,18 @@ describe('CKAN Extensions', () => {
             expect(installed_extensions).to.include('geodatagov_miscs');
             expect(installed_extensions).to.include('z3950_harvester');
             expect(installed_extensions).to.include('arcgis_harvester');
-            expect(installed_extensions).to.include('geodatagov_geoportal_harvester');
+            expect(installed_extensions).to.include(
+                'geodatagov_geoportal_harvester'
+            );
```

Comment:
Is there a line limit lint that put this on a new line?

Reply:
Yeah, this is Prettier's out of the box limit, which is set to 80 characters. I feel like I've seen a python limit somewhere, but can't recall where I've seen that. Do you know offhand?

Reply:
Yeah, the default python is also 80.  We set it to 128 in some of our repos, but yeah, I think somewhere around 120 should be a good limit.

Reply:
Changed my prettier config to 128 and repushing.